src/SQLite.Net/SQLite.Net2.csproj

```diff
@@ -32,15 +32,15 @@
     <Description>sqlite-net2 allows applications to manage data in SQLite databases using Entity Framework like queries, but much lighter</Description>
     <Product>$(AssemblyName) ($(TargetFramework))</Product>
     <VersionSuffix></VersionSuffix>
-    <PackageVersion>$(Version)$(VersionSuffix)</PackageVersion>
+    <PackageVersion>2.0.6.2</PackageVersion>
```

Comment:
where did this come from?

Reply:
This came from me. The version from the fork was 2.0.6, when we built it last it was 2.0.6.1, and this one i just bumped it one version number.